tests/test_subset.py

```diff
@@ -1502,6 +1502,46 @@ def test_temporal_merged_topex(self):
         assert (out_ds.time.values >= start_delta_dt).all()
         assert (out_ds.time.values <= end_delta_dt).all()
 
+    def test_get_time_epoch_var(self):
+        """
+        Test that get_time_epoch_var method returns the 'time' variable for the tropomi CH4 granule"
+        """
+        bbox = np.array(((-180, 180), (-90, 90)))
+        tropomi_file = 'S5P_OFFL_L2__CH4____20190319T110835_20190319T125006_07407_01_010202_20190325T125810_subset.nc4'
+
+        shutil.copyfile(os.path.join(self.test_data_dir, 'tropomi', tropomi_file),
+                        os.path.join(self.subset_output_dir, tropomi_file))
+
+
+        nc_dataset = nc.Dataset(os.path.join(self.subset_output_dir, tropomi_file), mode='r')
+
+        has_groups = bool(nc_dataset.groups)
+
+        # If dataset has groups, transform to work with xarray
+        if has_groups:
+            nc_dataset = subset.transform_grouped_dataset(nc_dataset, os.path.join(self.subset_output_dir, tropomi_file))
```

Comment:
You can just hardcode `has_groups` here since it's a test and you know whether or not `S5P_OFFL_L2__CH4____20190319T110835_20190319T125006_07407_01_010202_20190325T125810_subset.nc4` has groups